From 46e96a84ac187117332e99731b1e8f47bea9d8fc Mon Sep 17 00:00:00 2001
From: effects3d <effects3dee@gmail.com>
Date: Sun, 13 Sep 2026 13:43:46 +0200
Subject: [PATCH 05/10] =?UTF-8?q?Revert=20=E2=80=9Cloader:=20Install=20all?=
 =?UTF-8?q?=20fonts=20provided=20by=20Proton=20in=20the=20registry.?=
 =?UTF-8?q?=E2=80=9D?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

We’re only using the downstream fork of Wine used in Proton, so we can’t
install these fonts. This also reverts a regression with WebView2 not
functioning.

Source: Proton-Wine commit: 31b56a9a7b6b22e8c2e88776c2b7ed546d8fa84f
Signed-off-by: effects3d <effects3dee@gmail.com>
---
 loader/wine.inf.in | 14 ++------------
 1 file changed, 2 insertions(+), 12 deletions(-)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index af9fd075f7c..e957367c34b 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -540,26 +540,16 @@ HKLM,%FontSubStr%,"Times New Roman TUR,162",,"Times New Roman,162"
 HKLM,System\CurrentControlSet\Hardware Profiles\Current\Software\Fonts,"LogPixels",0x10003,0x00000060
 HKLM,%FontsNT%,"Arial (TrueType)",,"arial.ttf"
 HKLM,%FontsNT%,"Arial Bold (TrueType)",,"arialbd.ttf"
+HKLM,%FontsNT%,"Times New Roman (TrueType)",,"times.ttf"
 HKLM,%FontsNT%,"Courier New (TrueType)",,"cour.ttf"
-HKLM,%FontsNT%,"Courier New Bold (TrueType)",,"courbd.ttf"
 HKLM,%FontsNT%,"Lucida Console",,"lucon.ttf"
 HKLM,%FontsNT%,"Malgun Gothic (TrueType)",,"malgun.ttf"
-HKLM,%FontsNT%,"Marlett (TrueType)",,"marlett.ttf"
 HKLM,%FontsNT%,"Microsoft Sans Serif (TrueType)",,"micross.ttf"
-HKLM,%FontsNT%,"MS Gothic (TrueType)",,"msgothic.ttc"
-HKLM,%FontsNT%,"MS PGothic (TrueType)",,"msgothic.ttc"
-HKLM,%FontsNT%,"MS UI Gothic (TrueType)",,"msgothic.ttc"
 HKLM,%FontsNT%,"Microsoft YaHei (TrueType)",,"msyh.ttf"
 HKLM,%FontsNT%,"Microsoft YaHei Bold (TrueType)",,"msyhbd.ttf"
+HKLM,%FontsNT%,"MS Gothic (TrueType)",,"msgothic.ttc"
 HKLM,%FontsNT%,"Nirmala UI (TrueType)",,"nirmala.ttf"
 HKLM,%FontsNT%,"SimSun (TrueType)",,"simsun.ttc"
-HKLM,%FontsNT%,"NSimSun (TrueType)",,"simsun.ttc"
-HKLM,%FontsNT%,"Symbol (TrueType)",,"symbol.ttf"
-HKLM,%FontsNT%,"Tahoma (TrueType)",,"tahoma.ttf"
-HKLM,%FontsNT%,"Tahoma Bold (TrueType)",,"tahomabd.ttf"
-HKLM,%FontsNT%,"Times New Roman (TrueType)",,"times.ttf"
-HKLM,%FontsNT%,"Webdings (TrueType)",,"webdings.ttf"
-HKLM,%FontsNT%,"Wingdings (TrueType)",,"wingdings.ttf"
 HKCU,Software\Wine\Fonts\Replacements,"Palatino Linotype",,"Times New Roman"
 HKCU,Software\Wine\Fonts\Replacements,"Verdana",,"Times New Roman"
 HKCU,Software\Wine\Fonts\Replacements,"Segoe UI Semibold",,"Times New Roman"
-- 
2.55.0

